docs/sources/review/lint-prose/index.md

```diff
@@ -59,3 +59,72 @@ The task title contains a verb and an object. For example: "Create a dashboard".
 
 <!-- vale Grafana.Quotes = YES -->
 ```
+
+## Use Vale in VS Code
+
+You can use Vale to lint your current document in VS Code.
+
+1. Clone the [Writer's Toolkit](https://github.com/grafana/writers-toolkit/) repository.
+
+   ```bash
+   git clone git@github.com:grafana/writers-toolkit.git
+   ```
+
+1. Download and install [Vale](https://vale.sh/docs/vale-cli/installation/).
+
+   {{% admonition type="note" %}}
+   Refer to the Linux installation steps at [GitHub Releases](https://vale.sh/docs/vale-cli/installation/#github-releases). Verify that you are downloading the latest build of Vale for Linux.
+   You can use your Linux package manager to install Vale if it is included in your distribution repositories.
+   {{% /admonition %}}
+
+   {{< code >}}
+
+   ```linux
+   wget https://github.com/errata-ai/vale/releases/download/v2.28.0/vale_2.28.0_Linux_64-bit.tar.gz
+   mkdir bin && tar -xvzf vale_2.28.0_Linux_64-bit.tar.gz -C bin
+   export PATH=./bin:"$PATH"
+   ```
+
+   ```macos
+   brew install vale
```

Comment:
I think we should probably include a link to https://brew.sh/ in this section as well, people might not be familiar with it.

Reply:
Added as a Before you begin section